src/client/app/containers/AcctInputContainer.js

```diff
@@ -1,56 +1,60 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import Input from '../components/Input'
+
 import { validateInput } from '../services/'
 import { socket } from '../store/socket'
+
+import Input from '../components/Input'
+
 // action creators
 import { 
     submitAcct,
     validateClient
 } from '../store/reducers/acctInput'
+
 // action keys
 import {
     SUBMIT_ACCT_INPUT,
     RESPONSE_VALIDATE_CLIENT
 } from '../store/actions/'
 
-class AcctInput extends Component {
-    constructor(props) {
-        super(props),
-        this.state = {
-            inputValue: ''
-        }
-    }
-
-    handleInputChange(e) {
-        this.setState({ inputValue: validateInput(e.target.value) })
-    }
-
-    handleInputSubmit() {
-        this.setState( {inputValue: ""} )
-        this.props.submit(this.state.inputValue)
+class AcctInputContainer extends Component {
+    state = {
+        inputValue: ''
     }
-
+    
     componentWillMount() {
         socket.on(RESPONSE_VALIDATE_CLIENT, (data) => {
             this.props.validateClient(data)
         })
     }
```

Comment:
Any chance you can get rid of `componentWillMount` and make like a wrapper component with socket in it? I haven't used socket so I'm not sure if this is required or not. I just try not to use React's methods like this and `componentWillReceiveProps`, but if you're comfortable with it whatever